NotesBackEnd/tests/Notes.Api.Tests/Utils/EqualityConstraintExtension.cs

```diff
@@ -0,0 +1,10 @@
+﻿using NUnit.Framework.Constraints;
+
+namespace Notes.Api.Tests.Utils
+{
+    static class EqualityConstraintExtension

```

Comment:
It is more about a custom, however, always state modifiers explicitly. For people from different than C# domain might not be apparent, that this class is internal.